presto-raptor/src/main/java/com/facebook/presto/raptor/filesystem/HdfsModule.java

```diff
@@ -61,4 +66,17 @@ public Path createBaseLocation(StorageManagerConfig config)
     {
         return new Path(config.getDataDirectory());
     }
+
+    @Singleton
```

Comment:
The changes in raptor package could be a separate commit

Reply:
Talked to @shixuan-fan and agreed to keep the changes as it is.